datahub-frontend/build.gradle

```diff
@@ -46,4 +46,15 @@ graphqlCodegen {
 
 tasks.withType(Checkstyle) {
   exclude "**/generated/**"
+}
+
+checkstyleMain.source = "app/"
+
+distributions {
```

Comment:
Do you mind leaving a comment just discussing what this block achieves?

Reply:
This is basically a replacement of the distributions block in the play.gradle which no longer works with the new plugin, but yeah I can comment what it's doing: https://docs.gradle.org/current/userguide/distribution_plugin.html#sec:distribution_contents